build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -56,20 +59,20 @@ else
   select_probability="${SELECT_PROBABILITY:-0.1}"
 fi
 
-build/teamcity-roachtest-invoke.sh \
-  --metamorphic-encryption-probability=0.5 \
-  --metamorphic-arm64-probability="${ARM_PROBABILITY:-0.5}" \
-  --select-probability="${select_probability}" \
-  --use-spot="${USE_SPOT:-auto}" \
-  --cloud="${CLOUD}" \
-  --count="${COUNT-1}" \
-  --clear-cluster-cache="${CLEAR_CLUSTER_CACHE:-true}" \
-  --auto-kill-threshold="${AUTO_KILL_THRESHOLD:-0.05}" \
-  --parallelism="${PARALLELISM}" \
-  --cpu-quota="${CPUQUOTA}" \
-  --cluster-id="${TC_BUILD_ID}" \
-  --artifacts=/artifacts \
-  --artifacts-literal="${LITERAL_ARTIFACTS_DIR:-}" \
-  --slack-token="${SLACK_TOKEN}" \
-  --suite nightly \
-  "${TESTS}"
+#build/teamcity-roachtest-invoke.sh \
```

Comment:
Mistake?

Reply:
Nice catch! Forgot to uncomment after streamlining the process of testing in CI :)